commit d466d4b38d2cb46ccd7d87be3f9214ba88d4a79c
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-08-23T11:24:38+01:00 (7 weeks ago)
    
    OS-7967 unable to build vtfontcvt on older PIs due to queue.h changes

diff --git a/usr/src/tools/vtfontcvt/Makefile b/usr/src/tools/vtfontcvt/Makefile
index b8caefdb90..39c368b9a1 100644
--- a/usr/src/tools/vtfontcvt/Makefile
+++ b/usr/src/tools/vtfontcvt/Makefile
@@ -11,6 +11,7 @@
 
 #
 # Copyright 2017 Toomas Soome <tsoome@me.com>
+# Copyright 2019 Joyent, Inc.
 #
 
 CMDDIR=		$(SRC)/cmd/vtfontcvt
@@ -31,7 +32,13 @@ $(PROG):	$(OBJS)
 		$(LINK.c) $(OBJS) -o $@ $(LDLIBS)
 		$(POST_PROCESS)
 
+#
+# In order to build on older platforms, we specifically use the
+# source-tree version of queue.h. When building as part of
+# $SRC/cmd, we'll be using the $PROTO copy of queue.h anyway,
+# so this is a $SRC/tools workaround only.
+#
 %.o:		$(CMDDIR)/%.c
-		$(COMPILE.c) -o $@ $<
+		$(COMPILE.c) -I$(SRC)/uts/common -o $@ $<
 
 include		../Makefile.targ
